dna/database_to_json.py

```diff
@@ -106,6 +106,7 @@ def get_pipeline_run_info(self, pipeline_run):
         pipeline_run_info = {
                                 "pipeline": simple_pipeline,
                                 "pipeline_id": pipeline_id,
+                                "pipeline_run_id": pipeline_run_id,
```

Comment:
It seemed useful to keep this value here, just in case we wanted to more easily trace back a single instance in our metadataset to the full D3M DB pipeline run.